src/main/java/racingcar/domain/Distance.java

```diff
@@ -0,0 +1,52 @@
+package racingcar.domain;
+
+import java.util.Objects;
+
+public class Distance {
+
+    private int value;
+
+    private Distance(final int value) {
+        this.value = value;
+    }
+
+    public static Distance from(final int distance) {
+        return new Distance(distance);
+    }
+
```

Comment:
from 보다는 아래처럼 활용하면 어떨까요?? 그리고 0이라는 상수는 Distnace에서 관리하면 좋지 않을까요? 😀
```suggestion
    public static Distance initialize() {
        return new Distance(0);
    }

```